imagemounter/disk.py

```diff
@@ -471,6 +499,11 @@ def _mount_mmls_volumes(self):
                 volume.flag = 'unalloc'
             else:
                 volume.flag = 'alloc'
+                if ":" in slot:
+                    table, tslot = slot.split(':')
+                    volume.slot = int(table) * 4 + int(tslot)
```

Comment:
Hmm, partially :)

You can see an example of the use of collons here: http://digital-forensics.sans.org/blog/2010/01/11/using-image-offsets

00: ----- 0000000000 0000000000 0000000001 Primary Table (#0)
01: ----- 0000000001 0000000062 0000000062 Unallocated
02: 00:00 0000000063 0000064259 0000064197 DOS FAT12 (0x01)
03: 00:01 0000064260 0000273104 0000208845 DOS Extended (0x05)
04: ----- 0000064260 0000064260 0000000001 Extended Table (#1)
05: ----- 0000064261 0000064322 0000000062 Unallocated
06: 01:00 0000064323 0000273104 0000208782 DOS FAT16 (0x06)
07: 01:01 0000273105 0000433754 0000160650 DOS Extended (0x05)
08: ----- 0000273105 0000273105 0000000001 Extended Table (#2)
09: ----- 0000273106 0000273167 0000000062 Unallocated
10: 02:00 0000273168 0000433754 0000160587 NTFS (0x07)
11: 02:01 0000433755 0000481949 0000048195 DOS Extended (0x05)
12: ----- 0000433755 0000433755 0000000001 Extended Table (#3)
13: ----- 0000433756 0000433817 0000000062 Unallocated
14: 03:00 0000433818 0000481949 0000048132 Linux (0x83)
15: 03:01 0000481950 0000626534 0000144585 DOS Extended (0x05)
16: ----- 0000481950 0000481950 0000000001 Extended Table (#4)
17: ----- 0000481951 0000482012 0000000062 Unallocated
18: 04:00 0000482013 0000626534 0000144522 Linux Swap / Solaris x86 (0x82)
19: ----- 0000626535 0000629143 0000002609 Unallocated

Main problem is that only the primary table contains 4 records, all the other will contain at maximum 2, so the correct calculation is a little bit more complex and deserves a function on its own :)